internal/device/rapl_sysfs_power_meter.go

```diff
@@ -67,6 +67,11 @@ func (r *raplPowerMeter) Start(ctx context.Context) error {
 	return err
 }
 
+func (r *raplPowerMeter) Run(ctx context.Context) error {
+	<-ctx.Done()
+	return nil
+}
+
```

Comment:
since real and fake power meter do not actually `Run` and `Stop/Shutdown` , no need to implement `Runner` and `Shutdown. the `runServices` checks for interface implementation anyway.

Reply:
PowerMeters aren't services. It implements PowerMeter so these are specific to that interface.